### DISCUSSION TITLE
Point evaluations as right handside of variational forms

### DISCUSSION THREAD

**Discussion by Baumi3004** [firedrakeproject/firedrake#3183] [Firedrake support]
Hello, 
is it currently possible in firedrake, given a point $p$ in the domain, to solve a problems in the form
$$a(v,w) = f(v(p)) \text{ for all } v$$ I have tried to use Interpolators to VertexOnlyMeshes on the TestFunction $v$, but this does not work as the interpolator expects an actual function as an argument..
```
from firedrake import *
mesh = UnitSquareMesh(25, 25)

V = FunctionSpace(mesh, "CG", 1)
v = TestFunction(V)
w = TrialFunction(V)

points = [[0.1, 0.1], [0.2, 0.2], [0.9, 0.9]]
vom = VertexOnlyMesh(mesh, points)
P0DG = FunctionSpace(vom, "DG", 0)

f = Function(V)
solve( inner(v,w)*dx == interpolate(v, P0DG)*dx, f)
```

I would be really glad to get advice :) 
Kind regards
Lukas

**Comment by connorjward** [comment]
I'm not familiar enough with UFL to answer this for you. @ksagiyam @ReubenHill ?

**Comment by ReubenHill** [comment]
I believe this needs symbolic interpolation which will soon be available on master firedrake. For now, try out @nbouziani 's PR https://github.com/firedrakeproject/firedrake/pull/2297 and associated branch.
